serverjs/util.js

```diff
@@ -190,47 +190,48 @@ var exports = {
   newCard,
   addCardToCube,
   getCardImageURL,
-  arraysEqual: function(a, b) {
+  arraysEqual(a, b) {
```

Comment:
Doesn't have to be handled in this PR but we already have this function in `dist/utils/Util.js`.